dokuwiki-syntax/src/test/resources/dokuwiki10/specific/imagelink/imagelink2.test

```diff
@@ -0,0 +1,30 @@
+.#-----------------------------------------------------
+.input|dokuwiki/1.0
+.#-----------------------------------------------------
+[[http://php.net|{{ wiki:dokuwiki-128.png }}]]
+[[http://php.net|{{wiki:dokuwiki-128.png }}]]
+[[http://php.net|{{ wiki:dokuwiki-128.png}}]]
+.#-----------------------------------------------------
+.expect|event/1.0
+.#-----------------------------------------------------
+beginDocument
+beginParagraph
+beginLink [Typed = [false] Type = [url] Reference = [http://php.net]] [false]
+onImage [Typed = [false] Type = [attach] Reference = [dokuwiki-128.png]] [false] [[align]=[middle]]
+endLink [Typed = [false] Type = [url] Reference = [http://php.net]] [false]
+beginLink [Typed = [false] Type = [url] Reference = [http://php.net]] [false]
+onImage [Typed = [false] Type = [attach] Reference = [dokuwiki-128.png]] [false] [[align]=[right]]
+endLink [Typed = [false] Type = [url] Reference = [http://php.net]] [false]
+endParagraph
+beginParagraph
+beginLink [Typed = [false] Type = [url] Reference = [http://php.net]] [false]
+onImage [Typed = [false] Type = [attach] Reference = [:dokuwiki-128.png]] [false] [[align]=[left]]
+endLink [Typed = [false] Type = [url] Reference = [http://php.net]] [false]
+endParagraph
+endDocument
+.#-----------------------------------------------------
+.expect|xwiki/2.1
+.#-----------------------------------------------------
+[[[[image:dokuwiki-128.png||align="middle"]]>>http://php.net]][[[[image:dokuwiki-128.png||align="right"]]>>http://php.net]]
+
+[[[[image::dokuwiki-128.png||align="left"]]>>http://php.net]]
```

Comment:
Seems weird to me to have the first two in the same paragraph and the 3rd one on another. Is this really what you get on Dokuwiki ?

Reply:
Agreed, this is another bug. @nehagup I think you should also fix this and document this in the lira issue and commit message.